d.ts/eclint.d.ts

```diff
@@ -2,6 +2,8 @@
 /// <reference path="../typings/gulp-util/gulp-util.d.ts" />
```

Comment:
If TypeScript is updated, you can add this path to the `includes` section of `tsconfig.json` and you won't need this either. If you don't want to make this update, that's fine. I might go over it like I did with `linez` and fix it up later myself.